tooling/bundler/src/bundle/windows/templates/installer.nsi

```diff
@@ -347,7 +347,7 @@ Var AppStartMenuFolder
 !define MUI_FINISHPAGE_SHOWREADME_FUNCTION CreateDesktopShortcut
 ; Show run app after installation.
 !define MUI_FINISHPAGE_RUN "$INSTDIR\${MAINBINARYNAME}.exe"
-!define MUI_PAGE_CUSTOMFUNCTION_PRE SkipIfPassiveButUpdateShortcutIfUpdate
+!define MUI_PAGE_CUSTOMFUNCTION_PRE SkipIfPassive
```

Comment:
why was this changed?

Reply:
wasn't this change for migrating shortcuts? since we do it on install now, no need to do it here anymore

Reply:
No, this is for passive mode installer where it is not necessarily an update

Reply:
Actually I realize my implementation and name was wrong

Reply:
Let's ignore this and remove, my brain can't handle more NSIS this week

Reply:
isn't passive mode installer handled in install, line 592?

Reply:
yeah I think so, but I remember I had a case where ot wasn't covered by that and I added that code, I may have changed the code a bit after that. I really don't remember, let's just move on